monkey/monkey_island/cc/services/user_info.py

```diff
@@ -1,45 +1,52 @@
 
+__author__ = 'maor.rayzin'
 
-def extract_sam_secrets(mim_string, users_dict):
-    users_secrets = mim_string.split("\n42.")[1].split("\nSAMKey :")[1].split("\n\n")[1:]
 
-    if mim_string.count("\n42.") != 2:
-        return {}
+class MimikatzSecrets(object):
```

Comment:
The file is name user_info.py and has a single class called MimikatzSecrets. That doesn't add up. Rename the file

Reply:
Done